src/index.ts

```diff
@@ -18,6 +18,8 @@ import { notebookPlugin } from './pages/notebook';
 import { generateDefaultNotebookName } from './notebook-name';
 import { viewOnlyNotebookFactoryPlugin } from './view-only';
 
+import '../style/index.css';
+
```

Comment:
Please let me know if there is a better way to do this, as I could not make recursive `@import`s for CSS files work.